### DISCUSSION TITLE
Any plans to integrate OpenAI Codex / Copilot for the terminal?

### DISCUSSION THREAD

**Discussion by danielduckworth** [microsoft/terminal#13016] [Ideas]
Could be pretty useful.

There's a Zsh plugin for Codex for reference: https://github.com/tom-doerr/zsh_codex

**Comment by zadjii-msft** [comment]
I wouldn't say there's any official plans - more like hallway chatter at this point. @crutkas as an FYI (as the person we've been hallway chattering with 😉)

**Reply by DonLarry** [reply]
It looks like it's actually a feature now, but I can't find out some instructions or anything on how to get it (even tho I have copilot chat)

**Reply by zadjii-msft** [reply]
We're gonna track updates to this in #15000. There are definitely official plans, but it's not ready to ship quite yet. You can follow that thread for updates!
